lib/auth/helpers_mfa.go

```diff
@@ -221,8 +221,7 @@ func (d *TestDevice) solveRegisterWebauthn(c *proto.MFARegisterChallenge) (*prot
 	d.Key.PreferRPID = true
 
 	if d.passwordless {
-		d.Key.AllowResidentKey = true
-		d.Key.SetUV = true
+		d.Key.SetPasswordless()
```

Comment:
This here was missing a flag, plus at some point I added SetPasswordless() but probably missed this line.